pages/public_cloud/public_cloud_network_services/concepts-03-loadbalancer/guide.fr-fr.md

```diff
@@ -43,6 +79,66 @@ Une politique L7 est l'association entre une ou plusieurs règles L7 et un liste
 
 Lorsqu'une requête arrive sur un listener, les politiques sont évaluées dans l'ordre défini par l'attribut `position`. Lorsqu'une politique retourne `vrai`, l'action est effectuée et l'évaluation s'arrête. Si toutes les politiques retournent `faux`, la requête est alors envoyée au pool par défaut du listener.
 
+## Considérations réseau
+
+La configuration requise pour le Load Balancer Public Cloud varie en fonction du type de trafic entrant (privé ou public) et des adresses IP des membres (privées ou publiques). Les chapitres suivants présentent les 3 différents types d'architectures auxquels le Load Balancer peut répondre. Le tableau ci-dessous présente également les prérequis sur le réseau / les composants supplémentaires liés à chaque architecture.
+
+### Private to Private Load Balancer
+
+![Architecture du Load Balancer privée à privée](images/priv-to-priv.png){.thumbnail}
+
+Le trafic entrant provient d'un réseau privé et est routé vers des instances accessibles à partir de ce réseau privé. Dans ce cas, une adresse Floating IP ou une Public Gateway ne sont pas nécessaires.
+
+### Public to Private Load Balancer
+
+![Architecture du Load Balancer public-privé](images/pub-to-priv.png){.thumbnail}
+
+Le trafic entrant provient d'Internet et atteint une adresse Floating IP associée au Load Balancer. Les instances derrière le Load Balancer sont situées sur un réseau privé et n'ont pas d'IP publique, ce qui garantit qu'elles restent complètement privées et isolées d'Internet.
+
+### Public to Public Load Balancer
+
+![Public to Public Load Balancer architecture](images/pub-to-pub.png){.thumbnail}
+
+Le trafic entrant provient d'Internet et atteint une adresse Floating IP associée au Load Balancer. Les instances vers lesquelles le Load Balancer route le trafic sont accessibles avec une IP publique. Le Load Balancer utilise donc l’adresse Floating IP avec une sortie pour atteindre ces instances.
+
+### Prérequis réseau <a name="network-prerequisites"></a>
+
+| Type d'architecture | Réseau privé & sous-réseau | IP de passerelle définie dans les attributs de sous-réseau | Nombre d'adresses IP « libres » dans le sous-réseau | Composant Gateway & Floating IP \* |
+|---|---|---|---|---|
+|Private to Private | Obligatoire | Non requis | 3 ([src](/pages/public_cloud/public_cloud_network_services/known-limits)) | Non requis |
+|Public to Private | Obligatoire | Obligatoire | 5 ([src](/pages/public_cloud/public_cloud_network_services/known-limits)) | Obligatoire |
+|Public to Public | Obligatoire | Obligatoire | 5 ([src](/pages/public_cloud/public_cloud_network_services/known-limits)) | Obligatoire |
+
+\* : selon l'interface de contrôle que vous choisissez, soit ces composants seront créés pour vous (via l'espace client OVHcloud), soit vous devrez les créer spécifiquement (CLI OpenStack, Terraform ou GUI).
+
+## Statuts de fonctionnement et de provisioning <a name="operating-provisioning-status"></a>
```

Comment:
In the OVHCloud Control Plane, we decided to **not translate** provisioning / operating  status .

Reply:
Thanks for the information. I'll edit the FR version accordingly.